contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryLogicA2_3.sol

```diff
@@ -215,6 +224,7 @@ contract AutomationRegistryLogicA2_3 is AutomationRegistryBase2_3, Chainable {
     uint32 gasLimit,
     address admin,
     Trigger triggerType,
+    IERC20 billingToken,
```

Comment:
currently this function is in the common interface and will be breaking. but let me check if i can remove this function.

Reply:
I think you removed this, so going to mark this convo as resolved :)